test/NuGet.Core.Tests/NuGet.LibraryModel.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Reflection;
```

Comment:
I will remove and it see what happens, as long as it builds I don't think we need it